packages/smooth_app/lib/cards/product_cards/smooth_product_card_found.dart

```diff
@@ -1,3 +1,4 @@
+import 'package:flutter/cupertino.dart';
```

Comment:
This file/class name `SmoothProductCardFound` doesn't really match the thing it's doing, should we rename it ?

Reply:
This is above my pay grade, I have no idea :D

Reply:
hahahhaa ok I'll make that change :D